task.go

```diff
@@ -72,13 +72,11 @@ func (e *Executor) Run(ctx context.Context, calls ...taskfile.Call) error {
 	for _, call := range calls {
 		task, err := e.GetTask(call)
 		if err != nil {
-			e.ListTasks(FilterOutInternal(), FilterOutNoDesc())
-			return err
+			return fmt.Errorf("task: executor.Run(): executor.GetTask(): %v", err)
 		}
 
 		if task.Internal {
-			e.ListTasks(FilterOutInternal(), FilterOutNoDesc())
-			return &taskInternalError{taskName: call.Task}
+			return fmt.Errorf("task: executor.Run(): %v", &taskInternalError{taskName: call.Task})
```

Comment:
Same here

```suggestion
			return err
```

Reply:
I put back the original `&taskInternalError{taskName: call.Task}` because there is a unit test expecting an error there and `err` is nil, but it has the effect you were looking for.